packages/terra-popup/tests/wdio/popup-spec.js

```diff
@@ -18,10 +17,36 @@ describe('Popup', () => {
     beforeEach(() => browser.url('/#/tests/popup-tests/bounded'));
 
     Terra.should.beAccessible({ viewports });
-    Terra.should.matchScreenshot({ viewports });
     Terra.should.themeEachCustomProperty('#root', {
       '--terra-popup-content-header-border-bottom': '15px double red',
       '--terra-popup-content-close-icon': 'green',
     });
   });
+
+  Terra.viewports('tiny', 'medium').forEach((viewport) => {
+    describe('Arrow Popup closed', () => {
+      before(() => {
+        browser.url('/#/tests/popup-tests/arrow-closed');
+        browser.waitForVisible('#arrow-button');
+        browser.setViewportSize(viewport);
+        browser.click('#arrow-button');
+        browser.waitForVisible('#arrow-button');
+      });
+
+      Terra.should.matchScreenshot('#root');
+    });
+  });
+
+  Terra.viewports('tiny', 'medium').forEach((viewport) => {
+    describe('Bounded Popup closed', () => {
+      before(() => {
+        browser.url('/#/tests/popup-tests/bounded-closed');
+        browser.setViewportSize(viewport);
+        browser.waitForVisible('#bounded-button');
+        browser.click('#bounded-button');
+      });
+
+      Terra.should.matchScreenshot('#root');
```

Comment:
These tests are strange. They say popup closed, but they show a popup that is open.